v2/manifests/core-ons/the-train.yml

```diff
@@ -19,7 +20,6 @@ services:
       PORT:                        8084
       DP_COLOURED_LOGGING:         "true"
       DP_LOGGING_FORMAT:           "pretty_json"
-      PUBLISHING_THREAD_POOL_SIZE: 100
```

Comment:
This line has been removed as it was a duplicate configuration. If you expand the section above you will see the other one.